micronaut-webapp/build-docker-compose.sh

```diff
@@ -28,7 +28,7 @@ services:
     ports:
       - 8081:8081
     environment:
-      - TODOSERVICE_URL=http://todo-service:8080
+      - TODOSERVICE_URL=http://todo-service:8443
```

Comment:
https here probably right?
```suggestion
      - TODOSERVICE_URL=https://todo-service:8443
```

Reply:
Fixed